linera-service/src/linera/main.rs

```diff
@@ -1178,6 +1196,11 @@ fn main() -> Result<(), anyhow::Error> {
 }
 
 async fn run(options: ClientOptions) -> Result<(), anyhow::Error> {
+    if let ClientCommand::HelpMarkdown = options.command {
```

Comment:
nit: how about `if !matches!(options.command, ...) { ... }` ?

Reply:
Oh we can actually remove this entirely!